mountainduck/interface.md

```diff
@@ -32,6 +32,10 @@ Right-click on the taskbar and select *Taskbar setting → Notification area →
 
 `````
 
+```{tip}
+Using macOS, make sure that there is enought screen real estate left on the right side of the system status bar otherwise the Mountain Duck icon might not be visible.
+```
+
```

Comment:
```suggestion
```{admonition} macOS only
:class: tip

Make sure there is enough space left on right side of the notch or application menu items to make sure the _Mountain Duck_ is visible in the the system status bar
```
```

Reply:
ensure to add the dot at the end of the sentence and add the closing of the admonition below the sentence.